vm/src/frame.rs

```diff
@@ -862,6 +863,12 @@ impl Frame {
             }
             bytecode::Instruction::PopException {} => {
                 assert!(vm.pop_exception().is_some());
+                let block = self.pop_block();
+                assert!(block.is_some());
+                match block.unwrap().typ {
+                    BlockType::ExceptHandler {} => (),
+                    _ => assert!(false),
+                };
```

Comment:
I would change this into:
```rust
let block = self.pop_block.unwrap(); // this asserts that the block is_some.
if let BlockType::ExceptHandler = block.typ {
  ()
} else {
  panic!("Block type must be ExceptHandler here.")
}
```

Reasons for this:
- `assert!` might be ignored during release builds, where panic is not. I'm not sure about this.
- A match with one match, and an else case is clearer with the `if let` syntax in my opinion.

Reply:
Done :)